File: clients/java/examples/admin-drop-database.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.admin.AstraDBAdmin;
import java.util.UUID;

public class Example {
  public static void main(String[] args) {
    DataAPIClient client = new DataAPIClient("**APPLICATION_TOKEN**");

    AstraDBAdmin admin = client.getAdmin();

    admin.dropDatabase(UUID.fromString("**DATABASE_ID**"));
  }
}

File: clients/go/examples/admin-drop-database.go
package main

import (
	"context"
	"log"

	"github.com/datastax/astra-db-go/v2/astra"
	"github.com/datastax/astra-db-go/v2/astra/options"
)

func main() {
	ctx := context.Background()

	client := astra.NewClient(
		options.API().SetToken("**APPLICATION_TOKEN**"),
	)

	admin, err := client.Admin()

	if err != nil {
		log.Fatal(err)
	}

	err = admin.DropDatabase(ctx, "**DATABASE_ID**")

	if err != nil {
		log.Fatal(err)
	}
}
